.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,27 @@
+
+**What type of PR is this?**
+> Uncomment only one:
+>api-change
+>bug
+>cleanup
+>design
+>documentation
+>failing-test
+>feature
```

Comment:
Should have unit test and integration test types too

Reply:
done. Also added labels in the project for all categories.